md/release-notes.md

```diff
@@ -42,6 +42,10 @@ To be documented
 
 ### API removals
 
+#### BDM Api removal
+Since 7.0, BDM APIs and model classes in packages `com.bonitasoft.engine.bdm.*` where deprecated in favor of the Community version in packages `org.bonitasoft.engine.bdm.*`.  
+They are now removed in this version.
+
```

Comment:
What if I migrate from Bonita BPM 6.5 SP to Bonita 7.6?
Will I have anything to handle myself or can you tell me that a migration step is planned to help me handle this automatically?

Reply:
Nothing is planned for that.
A classical user using the Studio (and not direct Engine APIs) will not have any issue, as migration to 7.0.0 requires Sam to redeploy the BDM, so the changes will be handled for them.

Reply:
the https://documentation.bonitasoft.com/?page=migrate-from-an-earlier-version-of-bonita-bpm#toc1 migration page explicitly states that in this case, a migration step has to be managed using the Studio

Reply:
I suggest to add a comment to point to the migration doc page as a reminder

Reply:
I don't think this is very relevant, as someone migrating from 6.5.x to 7.6.0 will have to redeploy the BDM anyway, and will not have any problem.
More trouble would be for a customer calling Engine APIs from a client application and using deprecated model => for them, the classes will simply not be found. They will have to change the client code. Maybe we can enrich more this part...